src/current/_includes/cockroachcloud/metric-names-serverless.md

```diff
@@ -0,0 +1,35 @@
+The following is a list of available metrics for use in the Custom Metrics Chart page.
+
+### Changefeed 
+
+Name | Description
+-----|-----
+`changefeed.running` | Number of currently running changefeeds, including sinkless
+`changefeed.failures` | Total number of changefeed jobs which have failed
+`changefeed.error_retries` | Total retryable errors encountered by all changefeeds
+`changefeed.emitted_messages` | Messages emitted by all feeds
+`changefeed.emitted_bytes` | Bytes emitted by all feeds
+`changefeed.commit_latency` | Event commit latency: a difference between event MVCC timestamp and the time it was acknowledged by the downstream sink.  If the sink batches events,  then the difference between the earliest event in the batch and acknowledgement is recorded; Excludes latency during backfill
+`jobs.changefeed.currently_paused` | Number of changefeed jobs currently considered Paused
+`jobs.changefeed.protected_age_sec` | The age of the oldest PTS record protected by changefeed jobs.
+
+### Row-Level TTL
+
+Name | Description
+-----|-----
+`jobs.row_level_ttl.resume_completed` | Number of row_level_ttl jobs which successfully resumed to completion
```

Comment:
```suggestion
`jobs.row_level_ttl.resume_completed` | Number of `row_level_ttl` jobs which successfully resumed to completion
```
I think job names like this should probably be in code format, applies to other descriptions in this table as well

Reply:
updated.